src/cap-description.adoc

```diff
@@ -120,10 +120,13 @@ Execute Permission (X):: Allow instruction execution.
 
 [#asr_perm,reftext="ASR-permission"]
 Access System Registers Permission (ASR):: Allow read and write access to all
-privileged (M-mode and S-mode) CSRs with the following exceptions:
-. <<utid>>, <<utidc>>, <<stid>>, <<stidc>>, <<mtid>>, <<mtidc>> all require ASR
-access for writing and not for reading, as well as having a suitable privileged
-execution mode.
+privileged (M-mode and S-mode).
```

Comment:
Did you mean "privileged (...) CSRs"?

Reply:
I had thought that adding the M-Mode and S-Mode was unnecessary, but I guess it makes it more consistent.